code/modules/mob/living/carbon/human/life.dm

```diff
@@ -33,6 +33,11 @@
 	if(stat != DEAD)
 		//Stuff jammed in your limbs hurts
 		handle_embedded_objects()
+		
+	//yogs | start - bandage memes
+	if(stat != DEAD)
+		handle_bandaged_limbs()
+	//yogs | end
```

Comment:
It's ``yogs start - REASON``, not ``yogs | start - REASON``. Same applies to the ``yogs end``.